### DISCUSSION TITLE
README & app info should clarify that project is a fork of Tusky

### DISCUSSION THREAD

**Discussion by JJGadgets** [pachli/pachli-android#83] [General]
To maintain transparency to users, I think it is best to clearly state in the README, as well as respective stores' app listing descriptions, that this project is a fork of Tusky, as has been stated in the initial release blog post at https://pachli.app/pachli/2023/09/06/hello-pachli.html (the wording in which I think is about as clear and accurate as can be).

I felt reserved when I looked at the screenshots on the F-Droid listing and could not easily determine if this was a Tusky fork, if Pachli just happens to look similar in UI design, or at worst, a ripoff. Making the distinction clear would give me more confidence in what I'm about to install and use.

**Comment by nikclayton** [comment]
[edit to note that this was originally opened as an issue, I moved it to discussions since that felt more appropriate]

Hi, thanks for the feedback. I've moved this to GH discussions rather than issues as that feels like a slightly better place for this at the moment.

> Making the distinction clear would give me more confidence in what I'm about to install and use.

How?

I'm not necessarily disagreeing with you, but I'd like to dig in to this a little more. As the Pachli code base diverges further from the Tusky code the fact that it originated as a fork will become less and less relevant.

I'm also trying to avoid a front-and-centre presentation of the app as "It's like Tusky, but with all of the following bugs and irritating things fixed..." as I don't think that's respectful of the contributors to that project (and that will become out of date as the Tusky project make their own changes).

**Reply by Fangh** [reply]
I do agree it should be state it is a fork of Tusky. Maybe simply in the Readme. Because like I was opening a setting page and I was welcomed with the elephant picture that is the logo of Tusky and I was like "oh wtf ? Did I opened the wrong app ?"
